docs/resources/Audit_Log.md

```diff
@@ -0,0 +1,143 @@
+# Audit Logs Resource
+
+## Audit Logs
+
+Whenever an admin action is performed on the API, an entry is added to the respective guild's audit log. You can specify the reason by attaching the `X-Audit-Log-Reason` request header. This header supports url encoded utf8 characters.
+
+### Audit Log Object
+
+###### Audit Log Object Structure
```

Comment:
Uhhm. This would really be the /response/ structure. Not the object structure. Right?